### PR TITLE
Add missing Big Brother labels isError and version

### LINKED ISSUE #9: Add missing Big Brother labels isError and version
`servlet-monitor` needs to export the labels isError and version, according to the [big brother protocol](https://github.com/labbsr0x/big-brother). 

An example implementation can be found at the [express-monitor](https://github.com/labbsr0x/express-monitor) project.

### PR DESCRIPTION
**Pull request by werberson**
closes #9

### CODE REVIEW

File: src/main/java/br/com/labbs/monitor/filter/MetricsCollectorFilter.java [hunk 1]
```diff
@@ -43,39 +45,51 @@
...
-            if (!isEmpty(filterConfig.getInitParameter(DEBUG))) {
-                DebugUtil.setDebug(filterConfig.getInitParameter(DEBUG));
+            String debugParam = filterConfig.getInitParameter(DEBUG);
+            if (isNotEmpty(debugParam)) {
+                DebugUtil.setDebug(debugParam);
+            }
+            String versionParam = filterConfig.getInitParameter(VERSION);
+            if (isNotEmpty(versionParam)) {
+                this.version = versionParam;
             }
```
Comment (fabiotavarespr) [inline src/main/java/br/com/labbs/monitor/filter/MetricsCollectorFilter.java]: This parameter isn't in documentation, please add in doc or review the code

**Review by fabiotavarespr — CHANGES_REQUESTED** [review]
Please review in MetricsCollectorFilter.java everything about "private static final String VERSION = "version";"

File: src/main/java/br/com/labbs/monitor/filter/MetricsCollectorFilter.java (hunk 1 above)
Comment (werberson) [reply src/main/java/br/com/labbs/monitor/filter/MetricsCollectorFilter.java]: done

**Review by fabiotavarespr — APPROVED** [review]
LGTM

**Pull request merged**